sympy/geometry/ellipse.py

```diff
@@ -140,11 +140,14 @@ def __new__(
                 "eccentricity" must not be None.'''))
 
         if eccentricity is not None:
-            if hradius is None:
+            if eccentricity.is_real == True and eccentricity > 1:
```

Comment:
Can you add the tests?
I'd highly doubt that `eccentricity > 1` works because sympy may return unevaluated `Gt` that doesn't reduce to python booleans.
You may have to use `(eccentricity > 1) == True`

Reply:
Tests added